source/reference/program/mongotop.txt

```diff
@@ -174,22 +176,27 @@ This command produces the following output:
 
 .. code-block:: sh
 
-   connected to: 127.0.0.1
-
-                       ns       total        read       write           2012-08-13T15:45:40
-   test.system.namespaces         0ms         0ms         0ms
-     local.system.replset         0ms         0ms         0ms
-     local.system.indexes         0ms         0ms         0ms
-     admin.system.indexes         0ms         0ms         0ms
-                   admin.         0ms         0ms         0ms
-
-                       ns       total        read       write           2012-08-13T15:45:55
-   test.system.namespaces         0ms         0ms         0ms
-     local.system.replset         0ms         0ms         0ms
-     local.system.indexes         0ms         0ms         0ms
-     admin.system.indexes         0ms         0ms         0ms
-                   admin.         0ms         0ms         0ms
-
+                       ns    total    read    write          2014-12-19T15:32:01-05:00
+       admin.system.roles      0ms     0ms      0ms
+     admin.system.version      0ms     0ms      0ms
+                 local.me      0ms     0ms      0ms
+           local.oplog.rs      0ms     0ms      0ms
+   local.replset.minvalid      0ms     0ms      0ms
+        local.startup_log      0ms     0ms      0ms
+     local.system.indexes      0ms     0ms      0ms
+  local.system.namespaces      0ms     0ms      0ms
+     local.system.replset      0ms     0ms      0ms
+
+                       ns    total    read    write          2014-12-19T15:47:01-05:00
+       admin.system.roles      0ms     0ms      0ms
+     admin.system.version      0ms     0ms      0ms
+                 local.me      0ms     0ms      0ms
+           local.oplog.rs      0ms     0ms      0ms
+   local.replset.minvalid      0ms     0ms      0ms
+        local.startup_log      0ms     0ms      0ms
+     local.system.indexes      0ms     0ms      0ms
+  local.system.namespaces      0ms     0ms      0ms
+     local.system.replset      0ms     0ms      0ms
```

Comment:
local.system.indexes and local.system.namespaces should only appear in mmapv1? either want to note that or produce an alternate output example?